x/ccv/child/keeper/hooks.go

```diff
@@ -32,15 +38,18 @@ func (k Keeper) AfterValidatorDowntime(ctx sdk.Context, consAddr sdk.ConsAddress
 	k.slashingKeeper.SetValidatorSigningInfo(ctx, consAddr, signInfo)
 
 	// send packet to initiate slashing on the provider chain
-	k.SendPacket(
+	err = k.SendPenalties(
 		ctx,
 		abci.Validator{
 			Address: consAddr.Bytes(),
 			Power:   power,
 		},
+		valsetUpdate.ValsetUpdateId,
 		k.slashingKeeper.SlashFractionDowntime(ctx).TruncateInt64(),
 		signInfo.JailedUntil.UnixNano(),
 	)
+
+	panic(err)
```

Comment:
Why are you panicing here?

Reply:
Oh I missed to remove that one. It was for testing purpose. Thanks